docs/arcaflow/workflows/flow-control.md

```diff
@@ -107,3 +107,182 @@ outputs:
     my_plugin: !expr $.steps.my_plugin.outputs.success
     my_other_plugin: !expr $.steps.my_other_plugin.outputs.success
 ```
+
+### Dry Out with bindConstants()
+
+The builtin function [bindConstants()](expressions.md#functions) allows you to factor out any constant input variables to a foreach subworkflow. The input variable `name`'s value is repeated across each iteration in this input. This section will show you how to simplify it.
```

Comment:
We're still not clarifying that what we're about to see is the *complicated* version we're trying to simplify.

Since you have the H3 header here introducing the example, and an H4 header at 186 ("Dried out"), you should have another H4 header here introducing the archaic way to duplicate the key.

Reply:
kk I added that this is the 'more repetitive' section